### PR TITLE
LEAF-3505: Rename Workflows

### PR DESCRIPTION
**Pull request by raphaelfretter**
had to merge in 2390 for compatability in history log

### CODE REVIEW

**Review by aerinkayne — APPROVED** [review]

**Review by maxrichardjr — APPROVED** [review]

**Review by maxrichardjr — APPROVED** [review]

File: libs/logger/formatters/FormFormatter.php [hunk 1]
```diff
@@ -2,26 +2,30 @@
 
 namespace Leaf;
 
-class FormFormatter
-{
+class FormFormatter{
```
Comment (jampaul3) [inline libs/logger/formatters/FormFormatter.php:5]: Are you moving the { up to the same line as the class definition or is your IDE? I've noticed it in some instances but not all.

 PSR12 - 4.1 says

The opening brace for the class MUST go on its own line; the closing brace for the class MUST go on the next line after the body.

Opening braces MUST be on their own line and MUST NOT be preceded or followed by a blank line.

**Review by jampaul3 — APPROVED** [review]

File: LEAF_Request_Portal/api/controllers/WorkflowController.php
Comment (jampaul3) [inline LEAF_Request_Portal/api/controllers/WorkflowController.php:1]: Just curious if this is the only instance that modify is used for Workflow.

File: libs/logger/DataActionLogger.php [hunk 2]
```diff
@@ -156,4 +162,75 @@ function fetchLogItems($logResult){
 
         return $this->db->prepared_query($sqlFetchLogItems, $vars);
     }
+
+    private function findExternalValue($logData, $logType) {
```
Comment (Pelentan) [inline libs/logger/DataActionLogger.php:166]: type-hinting and return type, please.

File: libs/logger/formatters/FormFormatter.php (hunk 1 above)
Comment (raphaelfretter) [reply libs/logger/formatters/FormFormatter.php:5]: Thought I fixed this already. Will fix again

File: LEAF_Request_Portal/api/controllers/WorkflowController.php
Comment (raphaelfretter) [reply LEAF_Request_Portal/api/controllers/WorkflowController.php:1]: It is for now.

**Review by shaneodd — APPROVED** [review]

**Review by maxrichardjr — APPROVED** [review]

**Review by Pelentan — APPROVED** [review]

**Review by AmandaSumner — APPROVED** [review]

**Review by maxrichardjr — APPROVED** [review]

**Pull request merged**